finalPrep/README.md

```diff
@@ -23,52 +23,57 @@ Some things to consider as you refactor include...
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Front End curriculum site](https://frontend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [X] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Front End curriculum site](https://frontend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
+- [X] Study/Project work time (usually about 25 hours/week outside of class time)
+- [X] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+<img width="1440" alt="Screen Shot 2022-02-11 at 1 35 02 PM" src="https://user-images.githubusercontent.com/97919748/153666773-b1e2f374-1963-4f34-8493-22dc56577763.png">
+
 - `Add Week 2 Screenshot Here`
+<img width="1440" alt="Screen Shot 2022-02-11 at 1 35 12 PM" src="https://user-images.githubusercontent.com/97919748/153666831-253704ba-bc0c-487e-bced-1c9dfdb352ee.png">
+
 - `Add Week 3 Screenshot Here`
+<img width="1440" alt="Screen Shot 2022-02-11 at 1 35 20 PM" src="https://user-images.githubusercontent.com/97919748/153666868-7d533eeb-7fc4-4346-8a27-db26b6c6b37a.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [X] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [X] Add link to your gist here: https://gist.github.com/CorCanavan/4e540a38e77dfb59dfea549299d2aeca
```

Comment:
Love this, think you are set up well to start a strong relationship with your mentor!